src/switch_core.c

```diff
@@ -2452,6 +2453,13 @@ static void switch_load_core_config(const char *file)
 				} else if (!strcasecmp(var, "log-truncate")) {
 					int truncate = atoi(val);
 					switch_core_session_ctl(SCSC_LOG_TRUNCATE, &truncate);
+				} else if (!strcasecmp(var, "telnyx-rtp-poll-timeout-s") && !zstr(val)) {
```

Comment:
@jyotshna-telnyx discuss me that this is currently affecting one customer and they wanted to change this using a channel variable.